mongo/collection.go

```diff
@@ -290,7 +294,14 @@ func (coll *Collection) insert(ctx context.Context, documents []interface{},
 	return result, op.Execute(ctx)
 }
 
-// InsertOne inserts a single document into the collection.
+// InsertOne performs an insert operation (https://docs.mongodb.com/manual/reference/command/insert/) to insert a
+// single document into the collection.
+//
+// The document parameter must be the document to be inserted. It cannot be nil. If the document does not have an _id
+// field when transformed into BSON, one will be added automatically to the marshalled document. The original document
```

Comment:
Mention that you can retrieve the _id from the InsertOneResult

Reply:
done.